dlpy/utils.py

```diff
@@ -387,6 +387,20 @@ def get_imagenet_labels_table(conn):
     return conn.CASTable(temp_name)
 
 
+def get_user_defined_labels_table(conn, label_file_name):
```

Comment:
can you also change the get_imagenet_labels_table function to call this one.

Reply:
Yes - I assume the intention here is to have something like this:

def get_imagenet_labels_table(conn):
    temp_name = random_name('new_label_table', 6)

    filename = os.path.join('datasources', 'imagenet_labels.csv')
    project_path = os.path.dirname(os.path.abspath(__file__))
    full_filename = os.path.join(project_path, filename)
    
    return (get_user_defined_labels_table(conn, full_filename))